integrations/mysql.ini.tmpl

```diff
@@ -10,6 +10,9 @@ PASSWD   = {{TEST_MYSQL_PASSWORD}}
 SSL_MODE = disable
 
 [indexer]
+ISSUE_INDEXER_TYPE = elasticsearch
+ISSUE_INDEXER_CONN_STR = http://127.0.0.1:9200
```

Comment:
```suggestion
ISSUE_INDEXER_CONN_STR = http://elasticsearch:9200
```
127.0.0.1 isn't correct host because ES is in a docker container, so you need to use hostname of container.